examples/restinio/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2021 Niall Douglas
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/cpp-pm/gate
+include("../common.cmake")
+
+project(download-restinio)
+find_package(Threads REQUIRED)
+
+hunter_add_package(asio)
+find_package(asio CONFIG REQUIRED)
+
+hunter_add_package(http-parser)
+find_package(http-parser CONFIG REQUIRED)
+
+hunter_add_package(fmt)
+find_package(fmt CONFIG REQUIRED)
```

Comment:
see remark above about hunter dependencies, please hnunterize the restinio package

Reply:
It is fully hunterised.